science/coolprop-if97/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake  1.1
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           conflicts_build 1.0
+
+github.setup        CoolProp IF97 2.1.3 v
+github.tarball_from archive
+name                coolprop-if97
+revision            0
+license             MIT
+
+checksums           rmd160  fe452d42c8aabb0713c463b5be32a492ddb1121a \
+                    sha256  5d24a8ca0aeefb3a5085c9c10c92cf7b8e8c43a56953f5b5f7ced49cc03f6d1f \
+                    size    3893915
+
+maintainers         {juliensalort.org:mp @jsalort} openmaintainer
+categories-append   science
+
+description         Implementation of the IF97 formulation for the properties of pure water substance.
+long_description    Implementation of the IF97 formulation for the properties of pure water substance by \
+                    Ian Bell and the CoolProp team.
+homepage            http://www.coolprop.org
+
+cmake.build_type    Release
```

Comment:
does the default "MacPorts" as set by the cmake PG fail?

Reply:
Actually it doesn't. I have removed this line. Thanks.